test-cases/tests/installation/a26-verify-sendgrid-credentials-are-configured-properly.md

```diff
@@ -0,0 +1,48 @@
+---
+automation:
+  - INTLY-9950
+environments:
+  - osd-fresh-install
+estimate: 30m
+tags:
+  - per-release
+---
+
+# A26 - Verify SendGrid credentials are configured properly
+
+## Description
+
+This test case should verify that the OCM SendGrid Service has correctly setup SMTP credentials on the OpenShift cluster.
+
+## Steps
+
+1. Retrieve the SendGrid SMTP credentials from the RHMI Operator namespace
+
+   ```
+   # SMTP username, base64 encoded. This will be referred to as <username> from now on.
+   oc get secret redhat-rhmi-smtp -n redhat-rhmi-operator -o jsonpath='{.data.username}'
+
+   # SMTP password, base64 encoded. This will be referred to as <password> from now on.
+   oc get secret redhat-rhmi-smtp -n redhat-rhmi-operator -o jsonpath='{.data.password}'
+   ```
+
+2. Telnet to the SMTP service to ensure the credentials are correct
```

Comment:
this is the sendgrid way (see https://sendgrid.com/docs/for-developers/sending-email/getting-started-smtp/)

it also avoids sending a mail on every test run